### DISCUSSION TITLE
Togglable grid inventory

### DISCUSSION THREAD

**Discussion by le-kote** [space-wizards/space-station-14#23326] [Feature Request]
It would be good for some servers be able to not use new grid inventory, so maybe it should be togglable as the space wind

**Comment by CrigCrag** [comment]
can't downstream just not merge gridinv

**Comment by EmoGarbage404** [comment]
Downstreams are more than capable of managing their own content. It makes no sense for upstream to support two separate storage systems simultaneously when only one is going to be used.

**Comment by deathride58** [comment]
Gridinv as a system involves way too much backend work to support both the old system and gridinv simultaneously in a sane manner. Gridinv is backwards-compatible with the old system, but the old system is distinctly *not* forwards-compatible with gridinv. There is no sane way at all to make a cvar to toggle between gridinv and the old system.

That being said, however, it would be pretty easy for downstreams to build a remake of the old system that's properly compatible with gridinv's backend. But that likely isn't desirable here for a variety of reasons (most notably: maintenance overhead)
